src/e2e/specs/landing.spec.ts

```diff
@@ -5,117 +5,142 @@
 import { test, expect } from "../fixtures/basePage.js";
 import { defaultScreenshotOpts } from "../utils/helpers.js";
 
-test.describe(`${process.env.E2E_TEST_ENV} - Landingpage element verification`, () => {
+test.describe(`${process.env.E2E_TEST_ENV} - Verify the new Landing Page content`, () => {
```

Comment:
It's just going to be the landing page...

```suggestion
test.describe(`${process.env.E2E_TEST_ENV} - Verify the Landing Page content`, () => {
```